app/api/v1/utils/jwt_decorator.py

```diff
@@ -0,0 +1,25 @@
+# from flask import request
```

Comment:
> what does this file do?

Previously, when I was implementing JWT with pyJWT it was necessary to have a decorator to protect my routes but its not needed anymore since I started using flask-jwt-extended.